### ISSUE TITLE
Enumerable not implemented error in do_notify method

### ISSUE THREAD

**Issue by vysakh0** [honeybadger-io/honeybadger-elixir#35]
```
** (Protocol.UndefinedError) protocol Enumerable not implemented for "feedback.push.apple.com:2196"
    (elixir) lib/enum.ex:1: Enumerable.impl_for!/1
    (elixir) lib/enum.ex:140: Enumerable.count/1
    (elixir) lib/enum.ex:437: Enum.count/1
    (honeybadger) lib/honeybadger.ex:133: Honeybadger.do_notify/3
    (elixir) lib/task/supervised.ex:74: Task.Supervised.do_apply/2
    (stdlib) proc_lib.erl:239: :proc_lib.init_p_do_apply/3
Function: #Function<0.101430325/0 in Honeybadger.Logger.handle_event/2>
    Args: []
```

**Comment by rbishop** [comment]
This should be fixed in version 0.3.1 that we just released. Please update and let us know. Thank you!

**Comment by rbishop** [comment]
Closing for. @vysakh0 let me know if you're on 0.3.1 and still encountering this issue.
